docs/cave/d2d-gis-shapefiles.md

```diff
@@ -5,10 +5,12 @@ This section describes how to start the GIS application, configure the display a
 
 ## Display GIS Data
```

Comment:
Very beginning of the file needs a # Header to be consistent with all other pages.  Probably `# GIS Import`